gridcapa-swe-commons/src/main/java/com/farao_community/farao/gridcapa_swe_commons/hvdc/HvdcLinkProcessor.java

```diff
@@ -242,11 +242,20 @@ private static Optional<Generator> getOptionalGenerator(Network network, String
         return generator;
     }
 
-    private static Optional<Load> getOptionalLoad(Network network, String id,
+    private static Optional<Load> getOptionalLoad(Network network, Map<Integer, String> idsBypriority, TwoSides side,
                                                   Map<NetworkElement, Collection<String>> missingElementsMap) {
-        Optional<Load> load = Optional.ofNullable(network.getLoad(id));
+        Optional<Load> load = Optional.ofNullable(network.getLoad(idsBypriority.get(1)));
+        // only side one can have two ids
+        if (!load.isPresent() && side == TwoSides.ONE) {
+            Optional<Load> load2 = Optional.ofNullable(network.getLoad(idsBypriority.get(2)));
+
+            if (load2.isEmpty()) {
+                missingElementsMap.computeIfAbsent(NetworkElement.LOAD, k -> new HashSet<>()).add(idsBypriority.get(2));
+            }
+            return load2;
+        }
```

Comment:
Perhaps add a comment here to state that this is a temporary solution and explain a bit what it does, just like you have done in HvdcCreationParametersArrayDeserializer.java